### ISSUE TITLE
sqlite module: Notices thrown during WordPress import

### ISSUE THREAD

**Issue by OllieJones** [WordPress/performance#599]
## Bug Description ##

Notices thrown when using the WordPress post / page / media importer 

## Steps to reproduce

1. Make sure WP_DEBUG and WP_DEBUG_LOG are on.
1. On some working site with with a few posts or pages, do Tools > Export to get an XML file.
2. Go to a site with the sqlite module enabled, that is, one that's using the sqlite database.
3. Do Tools > Import, then download the WordPress importer.
4. Import your XML file.
5. Observe all the notices. 

I'm trying to track this down. Will keep you posted.

## Notices from debug.log

A lot of notices looking like this

```
[13-Dec-2022 15:42:18 UTC] PHP Notice:  Trying to get property 'ID' of non-object in /var/www/sqlite.lan.plumislandmedia.net/wp-includes/post.php on line 5405
[13-Dec-2022 15:42:18 UTC] PHP Stack trace:
[13-Dec-2022 15:42:18 UTC] PHP   1. {main}() /var/www/sqlite.lan.plumislandmedia.net/wp-admin/admin.php:0
[13-Dec-2022 15:42:18 UTC] PHP   2. WP_Import->dispatch() /var/www/sqlite.lan.plumislandmedia.net/wp-admin/admin.php:364
[13-Dec-2022 15:42:18 UTC] PHP   3. WP_Import->import() /var/www/sqlite.lan.plumislandmedia.net/wp-content/plugins/wordpress-importer/class-wp-import.php:65
[13-Dec-2022 15:42:18 UTC] PHP   4. WP_Import->process_posts() /var/www/sqlite.lan.plumislandmedia.net/wp-content/plugins/wordpress-importer/class-wp-import.php:89
[13-Dec-2022 15:42:18 UTC] PHP   5. wp_insert_post() /var/www/sqlite.lan.plumislandmedia.net/wp-content/plugins/wordpress-importer/class-wp-import.php:746
[13-Dec-2022 15:42:18 UTC] PHP   6. wp_transition_post_status() /var/www/sqlite.lan.plumislandmedia.net/wp-includes/post.php:4607

[13-Dec-2022 15:42:18 UTC] PHP Notice:  Trying to get property 'post_type' of non-object in /var/www/sqlite.lan.plumislandmedia.net/wp-includes/post.php on line 4706
[13-Dec-2022 15:42:18 UTC] PHP Stack trace:
[13-Dec-2022 15:42:18 UTC] PHP   1. {main}() /var/www/sqlite.lan.plumislandmedia.net/wp-admin/admin.php:0
[13-Dec-2022 15:42:18 UTC] PHP   2. WP_Import->dispatch() /var/www/sqlite.lan.plumislandmedia.net/wp-admin/admin.php:364
[13-Dec-2022 15:42:18 UTC] PHP   3. WP_Import->import() /var/www/sqlite.lan.plumislandmedia.net/wp-content/plugins/wordpress-importer/class-wp-import.php:65
[13-Dec-2022 15:42:18 UTC] PHP   4. WP_Import->process_posts() /var/www/sqlite.lan.plumislandmedia.net/wp-content/plugins/wordpress-importer/class-wp-import.php:89
[13-Dec-2022 15:42:18 UTC] PHP   5. wp_insert_post() /var/www/sqlite.lan.plumislandmedia.net/wp-content/plugins/wordpress-importer/class-wp-import.php:746
```

## Additional Context

 - PHP Version: 7.4.3
 - OS: Ubuntu 20.04
 - Browser: Firefox, Chrome
 - Plugin Version: 1.7.0
 - Device: Windows laptop with Ubuntu VM.
 - SQLite: 3.31.1

**Comment by mukeshpanchal27** [comment]
Thanks, @OllieJones for the report. I also reproduced a similar import issue in my setup when SQLite is enabled.

**Comment by aristath** [comment]
Pushed a fix in #600 👍
